pkg/addon-operator/operator.go

```diff
@@ -80,6 +85,10 @@ type AddonOperator struct {
 
 	// LeaderElector represents leaderelection client for HA mode
 	LeaderElector *leaderelection.LeaderElector
+
+	// CrdExtraLabels contains labels for processing CRD files
+	// like heritage=addon-operator
+	CrdExtraLabels map[string]string
```

Comment:
```suggestion
	CRDExtraLabels map[string]string
```

like APIServices and so on